x-pack/plugins/cases/docs/openapi/components/schemas/case_configure_customfields.yaml

```diff
@@ -0,0 +1,24 @@
+key:
+  description: >
+    A unique key for the custom field.
+    Must be lower case and composed only of a-z, 0-9, '_', and '-' characters.
+    It is used in API calls to refer to a specific custom field.
+  type: string
+  minLength: 1
+  maxLength: 36
+label:
+  description: The custom field label that is displayed in the case.
+  type: string
+  minLength: 1
+  maxLength: 50
+required:
+  description: >
+    Indicates whether the field is required.
+    If `false`, the custom field can be set to null when a case is created or updated.
```

Comment:
nit: It can also be omitted if not required.

Reply:
Thanks! Fixed in a5f32dd62c3ccc0d5142f046bcf04565d2b6fb04